modules/openapi-generator/src/main/resources/tiny-cpp-client/model-body.mustache

```diff
@@ -0,0 +1,177 @@
+
+{{#models}}{{#model}}
+#include "{{classname}}.h"
+
+using namespace Tiny;
+
+{{classname}}::{{classname}}()
+{
+    __init();
```

Comment:
Is there a reason for keeping constructor's body (and destructor's too) in a separate method?

Reply:
No, I think it was copied from another cpp class, where they used it outside the class